chroma-manager/tests/framework/integration/installation_and_upgrade/run_tests

```diff
@@ -26,6 +29,17 @@ TESTS_DIR="tests/integration/installation_and_upgrade/"
 
 trap "set +e; echo 'Collecting reports...'; scp root@$TEST_RUNNER:~/test_report*.xml \"$PWD/test_reports/\"" EXIT
 
+# first need to wipe the disks since this test doesn't use
+# setUp()
+pdsh -l root -R ssh -S -w "$(spacelist_to_commalist "${STORAGE_APPLIANCES[@]}")" "exec 2>&1; set -xe
```

Comment:
we do this within the test infrastructure , can we not use an existing method so we keep functionality in the same place?

Reply:
Done